build/Dockerfile.multi

```diff
@@ -0,0 +1,48 @@
+# https://goreleaser.com/docker/
+
```

Comment:
the Dockerfile in the `build` directory is used for goreleaser to build the release docker image. 

Did you try using the `Dockerfile` in the root for development? I've tried to keep it up to date, but if it doesn't work lmk and we can update it.

Reply:
ex: from the project root, im doing `docker build -t flipt/flipt:ot ` to test this out with the [tracing example](https://github.com/flipt-io/flipt/tree/main/examples/tracing)

Reply:
oh, nvm. I forgot thats actually for the devcontainer setup, allowing you to develop inside the container, not actually running flipt in the container

Reply:
What do you think about moving `Dockerfile.multi` to the root as just `Dockerfile` and moving the current devcontainer `Dockerfile` under the `.devcontainer/` folder.

This would also require updating the `devcontainer.json` to be:

```json
  "build": {
    "context": "..",
    "dockerfile": "Dockerfile"
  },
```

So that it would point to the local Dockerfile instead

Reply:
https://github.com/flipt-io/flipt/pull/1054